.release/security-scan.hcl

```
@@ -11,7 +11,16 @@ container {
 	triage {
 		suppress {
 			vulnerabilities = [
-				"CVE-2024-13176", # openssl@3.3.2-r4
+                # busybox@1.37.0-r12 https://nvd.nist.gov/vuln/detail/CVE-2025-46394
+                #
+                # Since we create the artifact .tar ourselves, this isn't a concern.
+				"CVE-2025-46394", 
```

I think the indentation here is mixing tabs and spaces so it looks a bit odd. Regarding the comment, I would clarify:
```suggestion
				# busybox@1.37.0-r12 https://nvd.nist.gov/vuln/detail/CVE-2025-46394
				# Boundary does not shell out to the busybox tar program.
				"CVE-2025-46394", 
```

Similarly for the others, we don't shell out to `gnupg` or `netstat`.

---

- Converted to tabs 14e90eab8e907db0895712e3a848b2a525487b1e
- Updated the comments 3a509f62f22efbae3cfe4eb18224ef7feb84a8bc